website/docs/components/tag/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,55 @@
+## Usage
+
+### When to use
+
+Use tags to indicate an object's categorization, i.e., for filtering.
+
+### When not to use
+
+- For static metadata, status, or to display a new feature, use [Badge](/components/badge/overview).
+- For collection enumeration or version number, use [BadgeCount](/components/badge-count/overview).
+
+### Layout
+
+- The tag component can be seen in several different layouts, but we commonly see them in a list or a group.
+- Tags will fill the parent container and then wrap, as necessary.
+
+#### List
+
+![Layout list example](/assets/components/tag/tag-layout-list.png)
+
+#### Group
+
+![Layout group example](/assets/components/tag/tag-layout-group.png)
+
+---
+
+### Spacing
+
+As a best practice, we recommend a vertical spacing of 12px and horizontal spacing of 8px between tags.
+
+![Spacing between tags](/assets/components/tag/tag-spacing.png)
+
+---
+
+### Leading vs. Trailing Icon
+
+
+The leading dismiss icon allows mouse users to clear multiple tags quicker, because they do not need to move their mouse, especially when tags have a more extended width.
```

Comment:
```suggestion
The leading dismiss icon allows mouse users to clear multiple tags quicker because they do not need to move their mouse, especially when tags have a more extended width.
```